### PR TITLE
LLVM and SPIRV-LLVM-Translator pulldown (WW20-21) 

### PR DESCRIPTION
**Pull request by vmaksimo**
LLVM: llvm/llvm-project@d30dfa867
SPIRV-LLVM-Translator: KhronosGroup/SPIRV-LLVM-Translator@c62ef5e

### CODE REVIEW

**Comment by vmaksimo** [comment]
Hi @steffenlarsen, could you please help to investigate a build failure (check-sycl target) on CUDA?
Example of failure: http://ci.llvm.intel.com:8010/#/builders/37/builds/8925/steps/16/logs/stdio

**Comment by steffenlarsen** [comment]
> Possibly, it can be related to google test update in LLORG: [d4d80a2](https://github.com/intel/llvm/commit/d4d80a2903c1d074008cac653cdb0b5fe39b8a00)

Good intuition! That was exactly it. https://github.com/steffenlarsen/llvm/commit/0331956375d6d86dbb016170419454faf0af5f4f did the trick on my machine.

**Comment by vmaksimo** [comment]
> Good intuition! That was exactly it. [steffenlarsen@0331956](https://github.com/steffenlarsen/llvm/commit/0331956375d6d86dbb016170419454faf0af5f4f) did the trick on my machine.

Thanks @steffenlarsen! This helped to fix the failure

File: clang/test/OpenMP/remarks_parallel_in_multiple_target_state_machines.c [hunk 1]
```diff
@@ -1,3 +1,7 @@
+// XFAIL: *
+// Failure is expected untill fixed in LLORG upstream.
```
Comment (bader) [inline clang/test/OpenMP/remarks_parallel_in_multiple_target_state_machines.c:2]: untill -> until

File: clang/test/OpenMP/remarks_parallel_in_target_state_machine.c [hunk 2]
```diff
@@ -1,3 +1,7 @@
+// XFAIL: *
+// Failure is expected untill fixed in LLORG upstream.
```
Comment (bader) [inline clang/test/OpenMP/remarks_parallel_in_target_state_machine.c:2]: untill -> until

**Review by bader — APPROVED** [review]
LGTM, except typo in the comments.

**Pull request merged**